backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-scheduling-backups-doc.adoc

```diff
@@ -61,10 +61,21 @@ spec:
     ttl: 720h0m0s
 EOF
 ----
+
 <1> `cron` expression to schedule the backup, for example, `0 7 * * *` to perform a backup every day at 7:00.
++
+[NOTE]
+====
+To schedule a backup at specific intervals, enter the `<duration in minutes>` in the following format:
+----
+  `schedule: "*/10 * * * *"`
+----
+Enter the minutes value between quotation marks(" ").
```

Comment:
Spacing and missing backticks:
```suggestion
Enter the minutes value between quotation marks (`" "`).
```
Although I don't know if the parenthetical is really necessary:
```suggestion
Enter the minutes value between quotation marks.
```
Up to you.